tests/test_api.py

```diff
@@ -407,6 +410,12 @@ def test_warm_start():
         model = estimator.model_
 
 
+@register_keras_serializable(name="CustomMetric")
+class CustomMetric(metrics_module.MeanAbsoluteError):
+    def __reduce__(self):
+        return metrics_module.deserialize, (metrics_module.serialize(self),)
```

Comment:
Why not implement a serializable metric in `utils`? Is this blocked by the TF pickle PR?

Reply:
Right. I don't think we should implement serializable helpers since (hopefully) it's going to be done upstream soon.